modules/ROOT/pages/prereq-hardware.adoc

```diff
@@ -9,9 +9,37 @@ To ensure the performance and stability of Anypoint Platform Private Cloud Editi
 [IMPORTANT]
 Before you install Anypoint Platform PCE, your infrastructure team must review each of the following sections and verify that your environment meets the stated disk and device requirements. If needed, contact your MuleSoft representative for assistance.
 
-== Memory and CPU Requirements
+There are 2 components of PCE:
+
+- Anypoint Platform Core
+- Monitoring Center Add-On
+
+To ensure platform performance, stability, and high availability, Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) supports two network configurations: 4-node and 7-node.
```

Comment:
Include version: Anypoint Platform Private Cloud Edition 3.0

Reply:
It's better to avoid specific versions when possible.